### ISSUE TITLE
V1.5.6 is slow

### ISSUE THREAD

**Issue by jivarson** [ironmansoftware/powershell-universal-legacy#85]
#### Description

Starting with 1.5.4, PU is reeally slow for me, barely loads admin- or dashboards.

#### Additional Information

in the log, most requests has an exectiontime of ~60 seconds. there are no [ERR], just [INFO]
a snippet for the log;
```
2020-12-13T21:22:08.9476751+01:00 0HM4VK9CFQEF8:00000006 [INF] DOMAIN\ADACCOUNT is part of role Administrator (c5ec6264)
2020-12-13T21:22:10.1187438+01:00 0HM4VK9CFQEFA:00000005 [INF] DOMAIN\ADACCOUNT is part of role Administrator (c5ec6264)
2020-12-13T21:22:10.9008576+01:00 0HM4VK9CFQEFD:00000005 [INF] DOMAIN\ADACCOUNT is part of role Administrator (c5ec6264)
2020-12-13T21:22:11.8472020+01:00 0HM4VK9CFQEFB:00000004 [INF] DOMAIN\ADACCOUNT is part of role Administrator (c5ec6264)
2020-12-13T21:22:12.6960479+01:00 0HM4VK9CFQEFE:00000005 [INF] DOMAIN\ADACCOUNT is part of role Administrator (c5ec6264)
2020-12-13T21:22:12.7703159+01:00 0HM4VK9CFQEFC:00000005 [INF] DOMAIN\ADACCOUNT is part of role Administrator (c5ec6264)
2020-12-13T21:22:16.3252747+01:00 0HM4VK9CFQEF6:00000005 [INF] DOMAIN\ADACCOUNT is part of role Administrator (c5ec6264)
2020-12-13T21:22:18.5080492+01:00 0HM4VK9CFQEF3:00000006 [INF] DOMAIN\ADACCOUNT is part of role Administrator (c5ec6264)
2020-12-13T21:22:23.6099913+01:00 0HM4VK9CFQEF4:00000005 [INF] DOMAIN\ADACCOUNT is part of role Administrator (c5ec6264)
2020-12-13T21:22:23.6117806+01:00 0HM4VK9CFQEF7:00000005 [INF] DOMAIN\ADACCOUNT is part of role Administrator (c5ec6264)
2020-12-13T21:22:24.9904464+01:00 0HM4VK9CFQEF5:00000005 [INF] DOMAIN\ADACCOUNT is part of role Administrator (c5ec6264)
2020-12-13T21:22:37.3339844+01:00 0HM4VK9CFQEF8:00000006 [INF] DOMAIN\ADACCOUNT is part of role GoliskaInternalUsers (ebd42241)
2020-12-13T21:22:37.3360460+01:00 0HM4VK9CFQEF8:00000006 [INF] Evaluting claims for DOMAIN\ADACCOUNT (1bffc0b6)
2020-12-13T21:22:37.4002310+01:00 0HM4VK9CFQEFA:00000005 [INF] DOMAIN\ADACCOUNT is part of role GoliskaInternalUsers (ebd42241)
2020-12-13T21:22:37.4012978+01:00 0HM4VK9CFQEFA:00000005 [INF] /admin/logo.ico (b0e61343)
2020-12-13T21:22:37.4013880+01:00 0HM4VK9CFQEFA:00000005 [INF] C:\Program Files (x86)\Universal\UniversalAutomation\logo.ico (ae8254ed)
2020-12-13T21:22:37.4014681+01:00 0HM4VK9CFQEFA:00000005 [INF] image/x-icon (44e8ed0d)
2020-12-13T21:22:37.4016635+01:00 0HM4VK9CFQEFA:00000005 [INF] Executing "Microsoft.AspNetCore.Mvc.PhysicalFileResult", sending file '"C:\Program Files (x86)\Universal\UniversalAutomation\logo.ico"' with download name '""' ... (5a6f99a2)
2020-12-13T21:22:37.4044525+01:00 0HM4VK9CFQEFA:00000005 [INF] Request finished in 58286.0184ms 200 image/x-icon (791a596a)
2020-12-13T21:22:38.9155721+01:00 0HM4VK9CFQEFD:00000005 [INF] DOMAIN\ADACCOUNT is part of role CUSTOMROLE (ebd42241)
2020-12-13T21:22:38.9173559+01:00 0HM4VK9CFQEFD:00000005 [INF] Executing endpoint '"Universal.Server.Services.VersionController.Get (Universal.Server)"' (500cc934)
2020-12-13T21:22:38.9174512+01:00 0HM4VK9CFQEFD:00000005 [INF] Route matched with "{action = \"Get\", controller = \"Version\"}". Executing controller action with signature "Microsoft.AspNetCore.Mvc.IActionResult Get()" on controller "Universal.Server.Services.VersionController" ("Universal.Server"). (122b2fdf)
2020-12-13T21:22:38.9175928+01:00 0HM4VK9CFQEFD:00000005 [INF] Executing ObjectResult, writing value of type '"System.String"'. (8a1b66c8)
2020-12-13T21:22:38.9177991+01:00 0HM4VK9CFQEFD:00000005 [INF] Executed action "Universal.Server.Services.VersionController.Get (Universal.Server)" in 0.3076ms (afa2e885)
2020-12-13T21:22:38.9178262+01:00 0HM4VK9CFQEFD:00000005 [INF] Executed endpoint '"Universal.Server.Services.VersionController.Get (Universal.Server)"' (99874f2b)
2020-12-13T21:22:38.9179026+01:00 0HM4VK9CFQEFD:00000005 [INF] Request finished in 59809.9843ms 200 text/plain; charset=utf-8 (791a596a)
```

#### Version Information

- Universal: 1.5.6
- Universal Hosting Method (IIS, Executable, docker etc): Kestrel
- PowerShell: 5.1
- Operating System: Windows Server 2019

#### Internal Issue

We'll fill this out when we create an internal issue in our repository.

**Comment by adamdriscoll** [comment]
Which authentication method are you using?

**Comment by adamdriscoll** [comment]
Resolved in 1.5.7
